scripts/gv_viewer.py

```diff
@@ -559,6 +560,28 @@ def fstr(label: str, e: enum.Enum):
     print(fstr('o', Controls.CYCLE_OBSERVATION))
 
 
+def set_observation_function(name: str, env: GridWorld):
```

Comment:
Very minor thing and silly thing, could we have env as first argument?  Just a stylistic choice.